docs/configuration/integrations/integrations-next/app-agent-receiver-config.md

```diff
@@ -0,0 +1,118 @@
++++
+title = "app_agent_config"
++++
+
+# app_agent_receiver_config
+
+The `app_agent_receiver_config` block configures the `app-agent-receiver`
+integration. This integration exposes a http endpoint that can receive telemetry
+from the [Grafana Javascript Agent](https://github.com/grafana/grafana-javascript-agent)
+and forward it to logs, traces or metrics backends.
+
+Full reference of options:
+
+```yaml
+  autoscrape:
+    # Enables autoscrape of integrations.
+    [enable: <boolean> | default = true]
+
+    # Specifies the metrics instance name to send metrics to. Instance
+    # names are located at metrics.configs[].name from the top-level config.
+    # The instance must exist.
+    #
+    # As it is common to use the name "default" for your primary instance,
+    # we assume the same here.
+    [metrics_instance: <string> | default = "default"]
+
+    # Autoscrape interval and timeout. Defaults are inherited from the global
+    # section of the top-level metrics config.
+    [scrape_interval: <duration> | default = <metrics.global.scrape_interval>]
+    [scrape_timeout: <duration> | default = <metrics.global.scrape_timeout>]
+
+  # Integration instance name
+  [instance: <string>]
+
+  # Traces instance to send traces to. This assumes that you have a traces config with such instance defined
+  [traces_instance: <string> | default = ""]
+
+  # Logs instance to send logs and exceptions to. This assumes that you have a logs
+  # config with the instance defined
+  [logs_instance: <string> | default = ""]
+
+  # Server config refres to the HTTP endpoint that the integration will be exposing
+  # to receive data from.
+  server:
```

Comment:
Does this inherit the TLS config (or new tls), or is this only HTTP?

Reply:
This is only HTTP for now.
If we have to support TLS, might need some help with implementing it